lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -200,6 +200,11 @@ class CategoryRenderer {
       descriptionEl.classList.add('lh-audit-group__description');
       auditGroupHeader.appendChild(descriptionEl);
     }
+    if (group.title === 'Metrics') {
```

Comment:
should move this over to `performance-category-renderer.js`. It'll have to do an extra step to re-find the header, but it would be good to keep specialized knowledge of the perf category encapsulated in that file if we can